src/Symfony/Component/Process/Process.php

```diff
@@ -331,7 +331,9 @@ public function start(callable $callback = null/*, array $env = array()*/)
         } else {
             $envPairs = array();
             foreach ($env as $k => $v) {
-                $envPairs[] = $k.'='.$v;
+                if (is_string($v)) {
```

Comment:
Should be more specific IMHO. Eg numbers should not be filtered. false !== ?

Reply:
Initially, this is the test I did to fix the problem, but after, I saw the test in the `getDefaultEnv()` method. I also prefer to use `false! ==` because it is more specific. I change that immediately.